"cosign initialize" doesn't work

I am trying to run `cosign` with a local TUF root. My understanding is that I should use the `initialize` command like this:
```
cosign initialize --root /local/tuf/repo/root.json
```
However this fails with:
```
Error: initializing root client: updating from remote TUF repository: updating tuf metadata: tuf: failed to decode timestamp.json: expired at 2021-12-02 15:41:56 +0000 UTC
main.go:46: error during command execution: initializing root client: updating from remote TUF repository: updating tuf metadata: tuf: failed to decode timestamp.json: expired at 2021-12-02 15:41:56 +0000 UTC
```
Running without `--root`:
```
cosign initialize    
Error: updating local metadata and targets: updating tuf metadata: tuf: failed to decode timestamp.json: expired at 2021-12-02 15:41:56 +0000 UTC
main.go:46: error during command execution: updating local metadata and targets: updating tuf metadata: tuf: failed to decode timestamp.json: expired at 2021-12-02 15:41:56 +0000 UTC
```

It appears it always fetch stuff from GCS which has an expired timestamp. I know there is a `--mirror` flag which can override the default GCS location but I don't want to put my stuff of GCS.

--- a/.github/workflows/sync.yml
+++ b/.github/workflows/sync.yml
@@ -1,0 +1,42 @@
+name: Sync Repository with GCS
+
+# Execute this on changes to repository/repository/
+on:
+  # When a new root is published.
+  push:
+    branches:
+      - main
+    paths:
+      - 'repository/repository/**'
+  workflow_dispatch:
+
+jobs:  
+  sync:
+    runs-on: ubuntu-20.04
+    permissions:
+      id-token: 'write'
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.x
+      - uses: 'google-github-actions/setup-gcloud@v0.2.1'
+        with:
+          project_id: project-rekor
+      # Setup OIDC->SA auth
+      - uses: 'google-github-actions/auth@v0.3.0'
+        id: auth
+        with:
+          token_format: 'access_token'
+          workload_identity_provider: 'projects/237800849078/locations/global/workloadIdentityPools/root-signing-pool/providers/sigstore-root'
+          service_account: 'sigstore-root-signing@project-rekor.iam.gserviceaccount.com'
+          create_credentials_file: true
+      - name: Login
+        run: |
+          gcloud auth login --brief --cred-file="${{ steps.auth.outputs.credentials_file_path }}"
+          gcloud auth list
+      # Sync  
+      - name: sync
+        run: gsutil cp -r repository/repository/* gs://sigstore-tuf-root/

